vega_sim/scenario/ideal_market_maker_v2/scenario.py

```diff
@@ -66,6 +67,7 @@ def __init__(
         proportion_taken: float = 0.8,
     ):
         self.num_steps = num_steps
+        self.random_agent_ordering = (random_agent_ordering,)
```

Comment:
Why a tuple?

Reply:
This looks like a fat finger at best. Will look.